server/app/controllers/admin/AdminImportController.java

```diff
@@ -308,6 +310,7 @@ public Result hxImportProgram(Http.Request request) {
    * neccessary question updates before saving the program
    */
   @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
+  @BodyParser.Of(LargeFormUrlEncodedBodyParser.class)
```

Comment:
This + the line change in `AdminImportViewPartial` is the fix for the large request body.